src/emm/cli/git_cli.py

```diff
@@ -129,14 +129,15 @@ def delete_branch(self, branch_name: str) -> None:
         for module in modules:
             rprint(f" - [yellow]{module}[/yellow]")
 
-    def update_branch(self, branch: str, rebase: bool) -> None:
+    def update_branch(self, branch: str, local: bool, rebase: bool) -> None:
         """
         Pull in changes from the specified branch.
 
         :param branch: Branch to pull changes from.
+        :param local: If True, update base local branch, else do nothing.
```

Comment:
I think this is the opposite right? If `local` is True, we will the local branch. If it's False, we'll pull from origin.

Reply:
Made the description more clear.